### ISSUE TITLE
Display current location on Map

### ISSUE THREAD

**Issue by Kletterer** [amosproj/amos-ss2021-3d-viewer#10]
**## User story**
1. As a user
2. I want to see my location on a map
3. So that I know where I am in the room and in which direction I am looking

**## Acceptance criteria**
* Map is shown in a small window in athe bottom right corner of the screen
* the location of the currently displayed panorama is shown
* (Zooming on the map & changing panoramas using the map is not part of this user story)
* classes ViewerImage are implemented as described in the API documentation

**## Definition of done**
* Added only after week 5
* The same for all features

**Comment by Mm24** [comment]
The feature is not ready for release. The map is shown overlayed with the panorama but the coordinate conversion with the prespective camera is not appropiate: 

- The map is not aligned qith the corner image
- The coordinates are not shown
- Integration with API is missing
